content/docs/contributing/release-process.md

```diff
@@ -314,7 +317,7 @@ page if a step is missing or if it is outdated.
         in the next steps.
 
         <div className="pageinfo pageinfo-info"><p>
-        🔰 Remember to keep open the terminal where you run <code>cmrel stage</code>. Its output will be used in the next step.
+        🔰 Remember to keep open the terminal where you run <code>cmrel makestage</code>. Its output will be used in the next step.
```

Comment:
Suggestion: remove as this actually is no longer needed (the output was the printed longer path of the staged artifacts)

Reply:
Haha I saw that I'd not updated the `stage` bit before and it blinded me to the fact that this wasn't needed any more! Good catch, will remove.